### DISCUSSION TITLE
How many times has the Mona Lisa been stolen? 🖼️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1217] [Galaxy Trivia] [answered]
**Bonus:** Guess the artist by the painting 🎨🖌️

<img width="719" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/8b673bb2-79fe-4054-9fa9-cecbc755451f">

**Comment by peterbaccaro** [comment, marked as answer]
The Mona Lisa has been stolen **once** on 21 August 1911 by an Italian Louvre employee who was driven to act by his Italian patriotism.

The painting you are referring to is Summer In Mountains II (2019) which is a painting by the artist **Serghei Ghetiu**.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/fc81012c-5ed5-4b50-86ca-95046898b257)

**Comment by Sajansharma0017** [comment]
The Mona Lisa has been stolen once but has been vandalized many times.
And the painting above is painted by the artist Serghei Ghetiu.

**Comment by CedericRomeyns** [comment]
1
